mods/ts/rules/defaults.yaml

```diff
@@ -508,6 +508,30 @@
 		AirborneUpgrades: airborne
 	ReturnOnIdle:
 
+^AircraftHusk:
+	RenderVoxels:
+	WithVoxelBody:
+	BodyOrientation:
+		QuantizedFacings: 0
+		CameraPitch: 90
+	Aircraft:
+		RepairBuildings:
+		RearmBuildings:
+	Health:
+		HP: 280
+	Armor:
+		Type: Heavy
+	HiddenUnderFog:
+		Type: CenterPosition
+	AutoTargetIgnore:
+	ScriptTriggers:
+	Tooltip:
+		GenericName: Destroyed Aircraft
+	FallsToEarth:
+		Spins: True
```

Comment:
I know we go over this every couple of months, but I can't remember. Wasn't the standard (which, admittedly, isn't really enforced) to use "True" and "False"?